src/mixins/_theme.scss

```diff
@@ -0,0 +1,86 @@
+@use "../design-tokens/colors.yml";
+@use "../design-tokens/opacity.yml";
+@use "../design-tokens/sizes.yml";
+
+/// Establish the minimal rules needed for a theme's root element, specifically
+/// a `background-color` and default text color. Without this, the theme root
+/// won't be visually offset from its parent and its contents may not be very
+/// readable.
+///
+/// @output `background-color` and `color` rules
```

Comment:
Nit: I'm not used to this comment style. Is this an SCSS convention or something?

Reply:
'Tis SassDoc! http://sassdoc.com/

Reply:
Gotcha, thanks!